cmd/review.go

```diff
@@ -73,6 +74,7 @@ var reviewCmd = &cobra.Command{
 
 		// Get summarize comment from diff datas
 		color.Cyan("We are trying to review code changes")
+		fmt.Println("11111111111111, ", cmd.Context())
 		resp, err := client.Completion(cmd.Context(), out)
 		if err != nil {
```

Comment:
Error generating comment for file cmd/review.go: Post "https://api.openai.com/v1/chat/completions": context deadline exceeded (Client.Timeout exceeded while awaiting headers)